addon/styles/addon.scss

```diff
@@ -12,148 +12,183 @@ $form-bkgrnd: rgba(0, 0, 0, 0);
 $modal-box-shadow: rgba(64, 64, 65, .5);
 $scroll-box-shadow: rgba(0, 0, 0, .25);
 
-.liquid-modal {
-  &.form-container {
-    .lm-container {
-      align-items: center;
-      justify-content: center;
-
-      > .frost-bunsen-form-dialog {
-        background-color: $form-bkgrnd;
-        border: 0;
-        max-width: none;
-        padding: 0;
-      }
-    }
-  }
-}
-
 .ps-container {
+  position: relative;
   max-height: calc(100vh - #{$modal-padding});
-  overflow: auto;
   padding: 0;
-  position: relative;
+  overflow: auto;
 
   .ps-scrollbar-y-rail {
-    margin-bottom: $action-bar-height + $info-bar-border-width;
     margin-top: $info-bar-height + $info-bar-border-width;
+    margin-bottom: $action-bar-height + $info-bar-border-width;
   }
 }
 
-.frost-modal-input {
-  background-color: $frost-color-white;
-  border: 0;
-  box-shadow: 0 0 20 $modal-box-shadow;
+.remodal {
```

Comment:
Similarly, I don't know if this can be scoped, but if so it would be good practice in case there are multiple types of modals (with different animations) in the system